temporal-sdk/src/main/java/io/temporal/activity/LocalActivityOptions.java

```diff
@@ -126,24 +147,28 @@ public LocalActivityOptions validateAndBuildWithDefaults() {
           startToCloseTimeout,
           localRetryThreshold,
           scheduleToCloseTimeout,
-          RetryOptions.newBuilder(retryOptions).validateBuildWithDefaults());
+          RetryOptions.newBuilder(retryOptions).validateBuildWithDefaults(),
+          doNotIncludeArgumentsIntoMarker);
     }
   }
 
   private final Duration scheduleToCloseTimeout;
   private final Duration localRetryThreshold;
   private final Duration startToCloseTimeout;
   private final RetryOptions retryOptions;
+  private boolean doNotIncludeArgumentsIntoMarker;
```

Comment:
nit. generally positive naming is preferable, can we call it `includeArgumentsIntoMarker` instead and flip the logic?

Reply:
I always follow the pattern that all properties defaults should match the type default. For a boolean the default value is false. That's why I named the property this way to make the false default value correct.